app/views/layout.mustache

```diff
@@ -45,6 +45,9 @@
     </PageMap>
     -->
     {{/has_pagemap}}
+    {{#include_survey}}
+    <script async="" defer="" src="//www.google.com/insights/consumersurveys/async_survey?site=5wdvu4vc5lhop3xbatxof6lzcm"></script>
```

Comment:
Why would we want to link to the **http** version of `www.google.com` if the pub server is running on `http`?

Even if the pub server is running on **non-https** the link should still be **https://www.google.com/...** I think.

So consider changing `//` with `https://`.